encoder_parameters: 
  elbow:
    counts_per_rotation: 1000
    offset: 500
  forearmRoll:
    counts_per_rotation: 1000
    offset: 500
  shoulder:
    counts_per_rotation: 1000
    offset: 500
  turntable:
    counts_per_rotation: 1000
    offset: 500
  wristPitch:
    counts_per_rotation: 1000
    offset: 500
  wristRoll:
    counts_per_rotation: 1000
    offset: 500
